### ISSUE TITLE
1.126 and 1.127 releases missing in maven central

### ISSUE THREAD

**Issue by schernov-xo** [hub4j/github-api#1084]
**Describe the bug**
The github project releases defines two latest versions: `v1.126` and `v1.127`, I've got e-mail notifications on both when they were published in [project releases](https://github.com/hub4j/github-api/releases).
Proof: https://mvnrepository.com/artifact/org.kohsuke/github-api
<img width="1113" alt="Screen Shot 2021-04-09 at 15 48 55" src="https://user-images.githubusercontent.com/41187778/114182276-1e9af200-994b-11eb-9cce-bb959b787af4.png">

Please ensure the artifact is available in central, the github release should be published not earlier.
Thanks.

**Comment by schernov-xo** [comment]
@bitwiseman it can be a failure of `release-drafter` bot

**Comment by bitwiseman** [comment]
```
[INFO] [INFO]  * Upload of locally staged artifacts finished.
[INFO] [INFO]  * Closing staging repository with ID "orgkohsuke-1280".
[INFO]
[INFO] Waiting for operation to complete...
[INFO] ........................................
[INFO]
[INFO] [INFO] Remote staged 1 repositories, finished with success.
[INFO] [INFO] Remote staging repositories are being released...
[INFO]
[INFO] Waiting for operation to complete...
[INFO] ..................
[INFO]
[INFO] [INFO] Remote staging repositories released.
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD SUCCESS
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  03:52 min
[INFO] [INFO] Finished at: 2021-04-09T01:30:18-07:00
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] Cleaning up after release...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  04:37 min
[INFO] Finished at: 2021-04-09T01:30:18-07:00
[INFO] ------------------------------------------------------------------------
```

**Comment by bitwiseman** [comment]
It looks like 1.126 did not actually get published but 1.127 did.   I don't know why 1.127 would not show up in mavencentral aside from delay in mirroring. 
![Screenshot (5)](https://user-images.githubusercontent.com/1958953/114207468-41a8b000-9911-11eb-9f93-09b718841439.png)

**Comment by schernov-xo** [comment]
Well, I can confirm that I successfully loaded dependency, but it still not listed in https://search.maven.org
But the `1.126` is not available (that's why I raised this issue).
Hence probably this issue may be closed.
